### PR TITLE
python3 guide

### PR DESCRIPTION
**Pull request by cancan101**

### CODE REVIEW

File: Python3.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Python 3 Guide
+_Python 3 for Python 2 devs._
+
+## Super calls
+
+Whenever possible, projects should take advantage of Python 3 features and syntax. For example prefer [new style super calls](https://docs.python.org/3.5/library/functions.html#super) to old stlye.
```
Comment (taion) [inline Python3.md]: "should take advantage of Python 3 features..." doesn't make sense in this context

Comment (taion) [inline Python3.md]: link to https://docs.python.org/3/library/functions.html#super

Comment (taion) [inline Python3.md]: "stlye"

File: Python3.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Prefer this:
+```python
+super().method()
+```
+
+to this:
+```python
+super(klass, self).method()
```
Comment (taion) [inline Python3.md]: better to use `MyClass` in example here

**Comment by taion** [comment]
and you should link this doc from the main README

**Comment by taion** [comment]
probably under https://github.com/4Catalyzer/python#python-3 – and add the high-level "prefer python 3 features" part there

**Comment by cancan101** [comment]
let's merge this and then I will update link in #4.

**Comment by taion** [comment]
Please don't merge un-approved PRs

**Pull request merged**